From: A. Maitland Bottoms <bottoms@debian.org>
Subject: debian armhf convert without neon

recent cmake and gcc 6 find arm_neon.h, but
then compilation of convert_with_neon.cpp fails.

--- a/host/lib/convert/CMakeLists.txt
+++ b/host/lib/convert/CMakeLists.txt
@@ -56,18 +56,18 @@
 ########################################################################
 # Check for NEON SIMD headers
 ########################################################################
-IF(CMAKE_COMPILER_IS_GNUCXX)
-    CHECK_INCLUDE_FILE_CXX(arm_neon.h HAVE_ARM_NEON_H)
-ENDIF(CMAKE_COMPILER_IS_GNUCXX)
-
-IF(HAVE_ARM_NEON_H AND (${CMAKE_SIZEOF_VOID_P} EQUAL 4))
-    ENABLE_LANGUAGE(ASM)
-
-    LIBUHD_APPEND_SOURCES(
-        ${CMAKE_CURRENT_SOURCE_DIR}/convert_with_neon.cpp
-        ${CMAKE_CURRENT_SOURCE_DIR}/convert_neon.S
-    )
-ENDIF()
+#IF(CMAKE_COMPILER_IS_GNUCXX)
+#    CHECK_INCLUDE_FILE_CXX(arm_neon.h HAVE_ARM_NEON_H)
+#ENDIF(CMAKE_COMPILER_IS_GNUCXX)
+#
+#IF(HAVE_ARM_NEON_H AND (${CMAKE_SIZEOF_VOID_P} EQUAL 4))
+#    ENABLE_LANGUAGE(ASM)
+#
+#    LIBUHD_APPEND_SOURCES(
+#        ${CMAKE_CURRENT_SOURCE_DIR}/convert_with_neon.cpp
+#        ${CMAKE_CURRENT_SOURCE_DIR}/convert_neon.S
+#    )
+#ENDIF()
 
 ########################################################################
 # Convert types generation
